src/commands.rs

```diff
@@ -63,6 +63,8 @@ pub struct InitMissionArgs {
     /// Description of mission
     #[arg(long)]
     pub message: Option<String>,
+    #[arg(long)]
+    pub properties: Option<serde_json::Value>,
```

Comment:
@ntlhui Can you weigh in on what this is for?

Reply:
This provides a machine readable map of additional pertinent values to this "mission".  For instance, this could be used to store config parameters, seed values, etc.